### ISSUE TITLE
README.md ... http://jipt.i18n.osticket.com/ ... 500 Internal Server Error

### ISSUE THREAD

**Issue by mlipok** [osTicket/osTicket#5863]
In
README.md 

There is link to 
http://jipt.i18n.osticket.com/ 

which will lead you to:
500 Internal Server Error

**Comment by JediKev** [comment]
@mlipok 

Yes, this is a known thing internally. This is on our list of things to do.

Cheers.

**Comment by mlocati** [comment]
Any news on this subject? And what's the difference between http://jipt.i18n.osticket.com/ and https://crowdin.com/project/osticket-official ?

**Comment by JediKev** [comment]
@mlocati 

Still on our todo list. 

`jipt.i18n.osticket.com` was an interactive inline editor meaning you can view and click around the helpdesk UI and translate things inline whereas `i18n.osticket.com` is just the main project page listing all the languages and lets you translate by file.

Cheers.
